CONTRIBUTING.md

```diff
@@ -0,0 +1,217 @@
+4. We avoid `this.` unless necessary.
+5. We always specify the visibility, even if it's the default (e.g.
+   `private string _foo' not `string _foo'). Visibility should be the first modifier (e.g.
+   `public abstract` not `abstract public`).
+6. Namespace imports should be specified at the top of the file, *outside* of `namespace` declarations should be sorted alphabetically.
+7. Avoid more than one empty line at any time. For example, do not have two
+   blank lines between members of a type.
+8. Avoid spurious-free spaces.
+   For example, avoid `if (someVar == 0)...`, where the dots mark the spurious-free spaces.
+   Consider enabling "View White Space (Ctrl+R, Ctrl+W)" or "Edit -> Advanced -> View White Space" if using Visual Studio to aid detection.
+9. If a file happens to differ in style from these guidelines (e.g. private members are named `m_member`
+   rather than `_member`), the existing style in that file takes precedence.
+10. We only use `var` when it's obvious what the variable type is (e.g. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`).
+11. We use language keywords instead of BCL types (e.g. `int, string, float` instead of `Int32, String, Single`, etc) for both type references as well as method calls (e.g. `int.Parse` instead of `Int32.Parse`). See issue [#13976](https://github.com/dotnet/runtime/issues/13976) for examples.
+12. We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.
+13. We use "`nameof(...)` "instead of ``` "..." ``` whenever possible and relevant.
+14. Fields should be specified at the top within type declarations.
+15. When including non-ASCII characters in the source code, use Unicode escape sequences (\uXXXX) instead of literal characters. Literal non-ASCII characters occasionally get garbled by a tool or editor.
+16. When using labels (for goto), indent the label one less than the current indentation.
+17. When using a single-statement if, we follow these conventions:
+    - Never use single-line form (for example: `if (source is null) throw new ArgumentNullException("source");`)
+    - Using braces is always accepted and required if any block of an `if`/`else if`/.../`else` compound statement uses braces or if a single statement body spans multiple lines.
+    - Braces may be omitted only if the body of *every* block associated with an `if`/`else if`/.../`else` compound statement is placed on a single line.
+
+An [EditorConfig](https://editorconfig.org "EditorConfig homepage") file (`.editorconfig`) has been provided at the root of the runtime repository, enabling C# auto-formatting conforming to the above guidelines.
+
+
+##### License information
+
+All source code files should start with this paragraph:
+
+```
+// Copyright 2021-2022 MONAI Consortium
```

Comment:
Developers are going to C&P this into every file, even if new.
New files should only have the year they were created, eg Copyright 2022. A file that was created in 2021 and modified this year would read Copyright 2021-2022. 
My suggestion would be to templatize the year in this file:  <<YEAR>>

Reply:
Updated